packages/terra-form-select/src/native-select/NativeSelectField.jsx

```diff
@@ -0,0 +1,204 @@
+   */
+  isIncomplete: PropTypes.bool,
+  /**
+   * Whether the field is displayed inline. Displays block by default.
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Whether the label is hidden. Allows hiding the label while meeting accessibility guidelines.
+   */
+  isLabelHidden: PropTypes.bool,
+  /**
+   * Whether the field displays as Invalid. Use when value does not meet validation pattern.
+   */
+  isInvalid: PropTypes.bool,
+  /**
+   * Additional attributes to spread onto the label.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  labelAttrs: PropTypes.object,
+  /**
+   * Set the max-width of a field using `length` or `%`.  Best practice recommendation to never exceed
+   * a rendered value of 1020px. _(Note: Providing custom inline styles will take precedence.)_
+   */
+  maxWidth: PropTypes.string,
+  /**
+   * Callback function triggered when the select value changes. function(value)
+   */
+  onChange: PropTypes.func,
+  /**
+   * The array of select options and opt groups.
+   */
+  options: PropTypes.arrayOf(PropTypes.oneOfType([optionPropType, optGroupPropType])),
+  /**
+   * Placeholder data. The presence of this data object is used as the indicator for whether or not a placeholder is used.
+   */
+  placeholder: PropTypes.shape({
+    /**
+     * Whether the placeholder is included as an option to clear selection.
+     */
+    allowClear: PropTypes.bool,
+    /**
+     * Optional override of the default display.
+     */
+    display: PropTypes.string,
+    /**
+     * Optional override of the default value.
+     */
+    value: PropTypes.string,
+  }),
+  /**
+   * Whether the field is required.
+   */
+  required: PropTypes.bool,
+  /**
+   * Additional attributes to spread onto the select.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  selectAttrs: PropTypes.object,
+  /**
+   * The Select identifier. Links the htmlFor of the field to the select identifier.
+   */
+  selectId: PropTypes.string.isRequired,
+  /**
+   * Whether to append the 'optional' label to a non-required field label.
+   */
+  showOptional: PropTypes.bool,
+  /**
+   * The value of the select. Can be a string or number.
+   */
+  value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+};
+
+const defaultProps = {
+  disabled: false,
+  hideRequired: false,
+  isIncomplete: false,
+  isInline: false,
+  isInvalid: false,
+  isLabelHidden: false,
+  required: false,
+  showOptional: false,
+};
+
+const NativeSelectField = ({
+  defaultValue,
+  disabled,
+  error,
+  help,
+  hideRequired,
+  isIncomplete,
+  isInline,
+  isLabelHidden,
+  isInvalid,
+  label,
+  labelAttrs,
+  maxWidth,
+  onChange,
+  options,
+  placeholder,
+  required,
+  selectAttrs,
+  selectId,
+  showOptional,
+  value,
+  ...customProps
+}) => {
+  const helpText = (
+    <span>
+      {help}
+    </span>
+  );
```

Comment:
This is probably short enough to be inlined, but it doesn't make a difference either way. 

```jsx
const helpText = <span>{help}</span>;
```

Or

```jsx
<Field
  help={<span>{help}</span>}
/>
```

Reply:
Updated.